modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro

```diff
@@ -175,6 +214,18 @@ definition {
 		}
 	}
 
+	macro searchKeys {
+		Click(locator1 = "CP2NavigationMenu#SEARCH_FILTER");
+
+		Type(
+			locator1 = "CP2NavigationMenu#SEARCH_FILTER",
+			value1 = "${searchinput}");
```

Comment:
Would you mind changing it to searchInput, please?

Reply:
I will do it